### DISCUSSION TITLE
Equivalent Chrome Settings in Qutebrowser

### DISCUSSION THREAD

**Discussion by mamsterdam11** [qutebrowser/qutebrowser#7297] [General]
Hi,

For a work-related website, I have to use kerberos for authentication. The way my company has set this up, it requires you to set up settings in your browser of choice. For example, below are the settings it requires for the website to work in Chrome:

```
$ defaults write com.google.Chrome AuthServerAllowlist "*.WEBSITE.COM,*.WEBSITE2.COM,localhost"
$ defaults write com.google.Chrome DisableAuthNegotiateCnameLookup -boolean true
```

The website now works when I use Chrome, but doesn't with qutebrowser. I'm wondering how I can make the above equivalent config changes in qutebrowser (or, the backend engine that qutebrowser is using), I couldn't really find anything equivalent at qute://settings/ . I don't have Chromium installed, but I was thinking somehow I can run the same commands for the Chromium-based backend of qutebrowser for this to work.

**Comment by toofar** [comment]
From grepping the webengine code it looks like the AuthServerAllowlist option maps to either `auth-server-allowlist` or `auth-server-whitelist` depending on your version (the latter is marked as deprecated). The are chrome/qt args.
No idea if you can change DisableAuthNegotiateCnameLookup. I guess it would be somewhere in data/webengine if it is anywhere.

**Reply by webknjaz** [reply]
I used `disable-auth-negotiate-cname-lookup=true` but I don't know how to check whether it works.

**Comment by The-Compiler** [comment]
It might also be worth trying to set those settings via macOS, but with `com.google.Chrome` replaced by `org.qt-project.Qt.QtWebEngineCore` or `org.qutebrowser.qutebrowser`. Don't think that will work though.
